VTEX - Marketplace APIs - Suggestions.json

```diff
@@ -2350,24 +2348,12 @@
                 }
             },
             "Images": {
-                "type": "array",
-                "description": "Array containing objects that represent the SKU's images. Each object should have a key representing the image file name and a value containing the URL of the image. The image must be sent through `https` protocol, otherwise it will not be rendered in VTEX Admin.",
-                "items": {
-                    "type": "object",
-                    "description": "An object that represents a single SKU image with a filename as the key and the image URL as the value.",
-                    "properties": {
-                        "imagem1.jpg": {
-                            "type": "string",
-                            "description": "The URL of the SKU's image. Must be an `https` URL.",
-                            "example": "https://imageurl.example"
-                        }
-                    }
-                },
-                "example": [
+                "type": "object",
+                "description": "Object that represent the SKU's images.",
```

Comment:
```suggestion
                "description": "Object that represents the SKU's images. For each image, it should have a key representing the image file name and a value containing the URL of the image. The image must be sent through `https` protocol, otherwise it will not be rendered in VTEX Admin.",
```